backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -445,6 +479,72 @@ public boolean addBreadcrumb(String message, Map<String, Object> attributes, Bac
         return database.getBreadcrumbs().addBreadcrumb(message, attributes, type, level);
     }
 
+    /**
+     * Enable backtrace metrics support
+     * @param settings  Backtrace metrics settings
+     */
+    public void enableMetrics(BacktraceMetricsSettings settings) {
```

Comment:
Why do I need to always provide metrics interface if we already know where to send requests? Why do we need a class instance to pass metrics settings?

Reply:
Metrics settings contain 5 parameters...then we will have functions which take 8+ params. In that case I think we tend to wrap params into an object.

Reply:
I will mark this as resolved @konraddysput